.travis.yml

```diff
@@ -1,9 +1,12 @@
 ---
 git:
   depth: 10
-language: generic
+language: node_js
 
-sudo: false
+node_js:
+  - "6"
+
+sudo: required # Until Yarn repo is added to apt-source-whitelist
 
 addons:
```

Comment:
Why not wait until you get the whitelist to change things over?
Won't dropping sudo make the CI runs slower?

Related to #1082.